packages/concerto-core/lib/introspect/metamodel.js

```diff
@@ -19,1069 +19,1069 @@ const ModelManager = require('../modelmanager');
 const Factory = require('../factory');
 const Serializer = require('../serializer');
 
+const metaModelCto = `/*
```

Comment:
Why changes to this file? That seems unrelated to `instanceof` ? I would remove those changes to avoid conflicts with the other PRs on the new parser.

Reply:
Ok, I will remove